integrations/cdi/hibernate-cdi/src/main/resources/META-INF/services/org.hibernate.engine.transaction.jta.platform.spi.JtaPlatformProvider

```diff
@@ -11,4 +11,5 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-io.helidon.integrations.cdi.hibernate.CDISEJtaPlatformProvider
\ No newline at end of file
+
+io.helidon.integrations.cdi.hibernate.HibernateJtaPlatformProvider
```

Comment:
As noted earlier in the review, renames like this should be reverted.

Reply:
Reverted